core/templates/dev/head/css/oppia.css

```diff
@@ -1487,7 +1487,7 @@ pre.oppia-pre-wrapped-text {
 */
 
 .oppia-rte-editor > p {
-  line-height: 1.5;
+  line-height: 1.846;
```

Comment:
Any reason for 1.846? Just curious....

Reply:
Yes, just check the line spacing (by inspect element) of the editor while editing and in the preview mode, these both inherit `1.846` value. This value comes from the `/third_party/generated/css/third_party.css`.

Reply:
Ah I see. Sounds good.

Reply:
Maybe we can have a comment on line 1488 explaining the reason of 1.846?

Reply:
Yes, right.